### PR TITLE
Created common header and footer

### PR DESCRIPTION
**Pull request by gonuguntla**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -25,4 +25,9 @@ def custom_devise_error_messages(error_msg_hash={})
 
     html.html_safe
   end
+
+
+  def info_site_home_page?
+    params["host"] == ENV['MLN_INFO_SITE_HOSTNAME'] && params["controller"] == "info_site" && params["action"] == "index"
```
Comment (daryanypl) [inline app/helpers/application_helper.rb:31]: where is MLN_INFO_SITE_HOSTNAME set up?

File: app/views/layouts/_navigation.html.erb [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-    <% end %>
-    <%= link_to "<i class='fa fa-question'></i> Help".html_safe, help_url %>
-    <% if user_signed_in? %>
-      <%= link_to "<i class='fa fa-user'></i> Account".html_safe, account_url %>
-    <% else %>
-      <%= link_to "<i class='fa fa-user'></i> Sign In".html_safe, new_user_session_path %>
-    <% end %>
+    <ul >
+      <li><%= link_to "<span>Home</span>".html_safe, root_url, :class=>"link-home", "aria-label" => "Home-Logo" %></li>
+      <li class="nav-hover-color"><%= link_to "Participating Schools", "//#{ENV['MLN_SETS_SITE_HOSTNAME']}:3000/schools" %></li>
```
Comment (daryanypl) [inline app/views/layouts/_navigation.html.erb:5]: I'm not sure :3000 should be hardcoded?

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (gonuguntla) [inline app/helpers/application_helper.rb:31]: @daryanypl This is already there in application.yml file. This is existing constant

File: app/views/layouts/_navigation.html.erb (hunk 2 above)
Comment (gonuguntla) [reply app/views/layouts/_navigation.html.erb:5]: Soeey i will remove it

Comment (gonuguntla) [reply app/views/layouts/_navigation.html.erb:5]: Sorry i will remove it

**Pull request merged**